sdk/playwrighttesting/microsoft-playwright-testing/src/core/playwrightService.ts

```diff
@@ -78,8 +119,25 @@ const getServiceConfig = (
     // mpt PAT requested and set by the customer, no need to setup entra lifecycle handlers
     validateMptPAT(exitWithFailureMessage);
   } else {
-    globalFunctions.globalSetup = require.resolve("./global/playwright-service-global-setup");
-    globalFunctions.globalTeardown = require.resolve("./global/playwright-service-global-teardown");
+    // If multiple global file is supported, append playwright-service global setup/teardown with customer provided global setup/teardown
+    if (isMultipleGlobalFileSupported) {
+      globalFunctions.globalSetup = [require.resolve("./global/playwright-service-global-setup")];
```

Comment:
is there a way we can move away from using "require" so that we can use ESM as our core soon?